app/serializers/spree/v2/storefront/store_serializer.rb

```diff
@@ -0,0 +1,21 @@
+module Spree
+  module V2
+    module Storefront
+      class StoreSerializer < BaseSerializer
+        include Spree::Api::V2::StoreMediaSerializerImagesConcern
+
+        set_type :store
+
+        attributes :name, :url, :meta_description, :meta_keywords, :seo_title, :default_currency,
+                   :default, :supported_currencies, :facebook, :twitter, :instagram, :default_locale,
+                   :customer_support_email, :description, :address, :contact_phone, :supported_locales,
+                   :allow_anonymous_question
+
+        has_many :menus
+        has_many :cms_pages
+
+        has_one :default_country, serializer: :country, record_type: :country, id_method_name: :default_country_id
+      end
+    end
+  end
+end
```

Comment:
store serializer is something we have in core right?
in that case we can only over ride and not redefine the file like this!

I was really surprised at first glance to see so many attributes defined in serializer